test.c

```diff
@@ -1839,6 +1866,32 @@ response_status_cb (http_parser *p, const char *buf, size_t len)
   return 0;
 }
 
+int
+chunk_header_cb (http_parser *p)
+{
+  assert(p == parser);
+  if (p->content_length == 0) {
+    // Terminating chunk.  Don't increment num_chunks in this case.
```

Comment:
OK, removed the redundant num_chunks in test declarations, see 33a79079b56af1390398d7862d2dba865e2e0002.